examples/wCCD/package.json

```diff
@@ -26,7 +26,7 @@
     },
     "scripts": {
         "build": "tsc && vite build",
-        "build:prod": "NETWORK='mainnet' vite build",
+        "build-prod": "NETWORK='mainnet' vite build",
```

Comment:
Changed this so the `build:prod` is unique for the browser-wallet, like `build:dev` is.